cli/deploy-managed-online-endpoint-mlflow-vnet.sh

```diff
@@ -0,0 +1,57 @@
+set -e
+
+# <set_environment_name>
+# export ENVIRONMENT_NAME="<ENVIRONMENT_NAME>"
+# # </set_environment_name>
+
+# #  environment name
+# export ENVIRONMENT_NAME=env-`echo $RANDOM`
+
+# # <create_environment>
+# az ml environment create --name $ENVIRONMENT_NAME -f endpoints/online/mlflow-vnet/environment/docker-context.yml
+# # </create_environment>
```

Comment:
@shivanissambare - 
Here we need to add - 
Step 1: docker building and pushing to acr

docker build -t mlflowbyoc:1 ./environment/docker-context
docker tag mlflowbyoc:1 {{acr}}.azurecr.io/mlflowbyoc:1
docker push {{acr}}.azurecr.io/mlflowbyoc:1

Step 2:
Step to create environment using image created in step 1